src/index.ts

```diff
@@ -23,9 +27,14 @@ async function run(params) {
     bootstrap();
   }
 
-  log.debug(`Start command ${params._[0]}`);
-  await cmd(params);
-  log.debug(`Finished command ${params._[0]}`);
+  log.debug(`Start command ${command}`);
+  if (['deploy', 'import'].includes(command) && 'input_file' in params) {
```

Comment:
Breaking out the one-liner `await cmd(params)` into a few more lines making it explicitly clear which commands are being executed.

Also, the `'input_file' in params` and `'output_folder' in params` conditionals are necessary to convince the compiler that the correct parameter type is getting passed through.